SharePointFramework/PortfolioWebParts/src/components/LatestProjects/LatestProjects.module.scss

```diff
@@ -1,22 +1,24 @@
 .root {
   .container {
-    padding: 5px;
-    margin: 3px 0 0 0;
-
     .projectItem {
-      padding: 15px 2px;
-      border-bottom: 1px solid rgb(234, 234, 234);
-      .itemContainer {
-        height: 100%;
-        -webkit-box-sizing: border-box;
-        box-sizing: border-box;
-        overflow: hidden;
-        position: relative;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      padding: 12px 0;
+      height: 100%;
+      -webkit-box-sizing: border-box;
+      box-sizing: border-box;
+      overflow: hidden;
+      position: relative;
+
+      .projectInformation {
+        padding-left: 12px;
```

Comment:
This is added even when logo is not enabled, which causes some wierd UI. Will fix in next commit @Remi749